.template/problem.tex

```diff
@@ -4,19 +4,83 @@
 
 \makeheader
 
-\placeholder{Problem description}
+ProPong
+
+Since you heard about a trend at your University to make highly competitive games out of seemingly simple drinking games you created a new upgraded version of the famous Beerpong game. Unlike in the original game, the rules here are a bit different to eliminate every chance of winning the game just by luck. 
+
+The setup consists of a long table with a team on each of the two shorter sides of it. As in the original game, teams take turns throwing a ping pong ball at some cups placed on the table. Every time a team hits a cup the team owning that cup has to drink its content and take the cup from the table. This also holds if someone manages to hit his/her own cup. The team who first manages to throw a ball into all the cups belonging to their opponents wins the game. 
+
+In contrast to the silly luck-based version of the game, the cups are not placed in a triangle on each side of the table but rather in the shape of a rectangle in the table's center. This rectangle contains  $n \cdot m$ cups of which one half belongs to you and the other half to your opponents. The cups are placed in a random permutation which will be changed after each match so that after an arbitrary number of k games the effects of this randomness will be insignificant. 
```

Comment:
The cups being randomly changed after each match might be confusing because it's not relevant to the problem.